src/credssp/sspi_cred_ssp/tls_connection.rs

```diff
@@ -0,0 +1,220 @@
+use std::io::{Read, Write};
```

Comment:
For my information, do you think this module can compile for WASM? I don’t think rustls is fully WASM-compatible. I’m not sure [wasm-testcompile](https://github.com/Devolutions/sspi-rs/tree/master/tools/wasm-testcompile) is triggering code path in this module. Do you think you could extend it? (Unless it doesn’t make sense for us to support TSSSP in IronRDP of course.)

Reply:
It doesn't make sense to support TSSSP in `IronRDP`. TSSSP is only expected to be used on _Windows_ when `sspi-rs` is used with `mstsc`.

Reply:
@CBenoit as long as we can build this for Windows only without breaking wasm-builds with TSSSP disabled and other platforms, we're good. it's only going to be used for mstsc.exe injection for now

Reply:
Let me make allow compiling it only on `Windows`

Reply:
Great, thank you!